content/01-getting-started/02-setup-prisma/03-start-from-scratch-prisma-migrate.mdx

```diff
@@ -393,6 +393,7 @@ This command reads your Prisma schema and generates your Prisma Client library i
 
 The `@prisma/client` node module references a folder named `.prisma/client`, which contains your unique, generated Prisma client:
 
+// THIS IS BROKEN IN PROD
 ![The .prisma and @prisma folders](generating-prisma-client/prisma-client-node-module.png)
```

Comment:
@nikolasburk @mhwelander - I just noticed this link is broken in production. I think we should try to fix this ASAP (see screenshot in PR description for reference)